server/controllers/chartsController.js

```diff
@@ -56,14 +56,15 @@ const postProcessData = (data, studyTotals) => {
   }
 
   for (const [study, values] of processedDataBySite) {
-    const studySectionTotals = studyTotals[study]
-    const count = studySectionTotals.targetTotal
-      ? studySectionTotals.targetTotal - studySectionTotals.count
-      : 0
-    const percent = studyCountsToPercentage(
-      count,
-      studySectionTotals.targetTotal ?? studySectionTotals.count
+    const { targetTotal, count: currentSiteCount } =
+      studyTotals[study].studySectionTotals
+    const isTargetPositive = targetTotal && targetTotal > currentSiteCount
```

Comment:
this isn't determining if the target is positive but instead if it is greater than the count.

Reply:
haha, thank you for bringing that up. I was considering isTargetGreaterThanCount but it looks to long, what are your thoughts on a name?